### PR TITLE
'VBM-77: change user's access for navigation

### PR DESCRIPTION
**Pull request by Val-M9**

### CODE REVIEW

File: packages/mobile/src/navigation/navigation.tsx [hunk 1]
```diff
@@ -30,47 +26,32 @@ const accountScreenOptions: NativeStackNavigationOptions = {
...
-          <NativeStack.Screen
-            name={RootScreenName.MAIN}
-            component={MainNavigation}
+      <NativeStack.Group>
+        <NativeStack.Screen
+          name={RootScreenName.MAIN}
+          component={MainNavigation}
+        />
+        <NativeStack.Group screenOptions={accountScreenOptions}>
+          <Stack.Screen
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/navigation.tsx:37]: Groups are not supposed to be nested

File: packages/mobile/src/navigation/welcome/welcome.navigation.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {
+  createNativeStackNavigator,
+  NativeStackNavigationOptions,
+} from '@react-navigation/native-stack';
+import { RootScreenName } from '~/common/enums/enums';
+import { RootNavigationParamList } from '~/common/types/types';
+import { Auth, Welcome } from '~/screens/screens';
+
+const WelcomeNavigation = () => {
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/welcome/welcome.navigation.tsx:10]:
```suggestion
const WelcomeNavigation: React.FC = () => {
```

File: packages/mobile/src/navigation/welcome/welcome.navigation.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const NativeStack = createNativeStackNavigator<RootNavigationParamList>();
+  const welcomeScreenOptions: NativeStackNavigationOptions = {
+    headerShown: false,
+  };
+
+  return (
+    <NativeStack.Navigator screenOptions={welcomeScreenOptions}>
+      <NativeStack.Group>
```
Comment (AleksandrTermenzhy) [inline packages/mobile/src/navigation/welcome/welcome.navigation.tsx:18]: You can skip using group here as there is only one

**Review by AleksandrTermenzhy — CHANGES_REQUESTED** [review]

File: packages/mobile/src/navigation/navigation.tsx (hunk 1 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/navigation.tsx]: fixed

File: packages/mobile/src/navigation/welcome/welcome.navigation.tsx (hunk 2 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/welcome/welcome.navigation.tsx]: fixed

File: packages/mobile/src/navigation/welcome/welcome.navigation.tsx (hunk 3 above)
Comment (Val-M9) [reply packages/mobile/src/navigation/welcome/welcome.navigation.tsx]: fixed

**Comment by Val-M9** [comment]
[Відео без назви - Screen Recording - 25.08.2022, 13_06_47.webm](https://user-images.githubusercontent.com/71255723/186638223-2924b415-7971-4b6d-9b0a-135cbcd39c54.webm)

**Review by AleksandrTermenzhy — APPROVED** [review]

**Pull request merged**